controllers/predictor_controller.go

```diff
@@ -239,6 +240,33 @@ func validatePredictor(predictor *api.Predictor) string {
 		return "Only one of spec.schemaPath and spec.storage.schemaPath can be specified"
 	}
 
+	if storage.StorageSpec.Parameters != nil {
```

Comment:
I actually think we can probably omit this whole section. If the PVC isn't present it will cause the predictor load to fail (with an informative message from the puller).